core/src/main/scripts/importer/validateData.py

```diff
@@ -1192,6 +1181,7 @@ class ClinicalValidator(Validator):
     REQUIRE_COLUMN_ORDER = False
     PROP_IS_PATIENT_ATTRIBUTE = None
     NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", ""]
```

Comment:
I thought in a conversation yesterday you wanted us to replace 'null' with 'na' (at least for expression data).  Is 'na' missing from this list?